adapters/libev.h

```diff
@@ -103,13 +104,41 @@ static void redisLibevDelWrite(void *privdata) {
     }
 }
 
+static void redisLibevStopTimer(void *privdata) {
+    redisLibevEvents *e = (redisLibevEvents*)privdata;
+    struct ev_loop *loop = e->loop;
+    ((void)loop);
+    ev_timer_stop(EV_A_ &e->timer);
+}
+
 static void redisLibevCleanup(void *privdata) {
     redisLibevEvents *e = (redisLibevEvents*)privdata;
     redisLibevDelRead(privdata);
     redisLibevDelWrite(privdata);
+    redisLibevStopTimer(privdata);
     free(e);
 }
 
+static void redisLibevTimeout(EV_P_ ev_timer *timer, int revents) {
+    ((void)revents);
+    redisLibevEvents *e = timer->data;
+    redisAsyncHandleTimeout(e->context);
+}
+
+static void redisLibevScheduleTimer(void *privdata, struct timeval tv) {
```

Comment:
@michael-grunder please note that currently only `libevent.h` implements a timer and it uses the name `redisLibeventSetTimeout()`, please consider adopting the same naming convention across adapters.

Reply:
Updated